composites/enforce-pmc-scripts/enforce-pmc-scripts.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -e
+
+HAS_ERROR=""
+NEED_UPDATE_PACKAGE_JSONS=()
+NEW_JS_FILES=()
+
+find_package_json() {
+  if [[ -f "$1/package.json" ]]; then
+    if [[ -n $( grep "@wordpress/scripts" $1/package.json ) || -n $( grep "@penskemediacorp/wordpress-scripts" $1/package.json ) ]]; then
+      echo "$1/package.json"
+    fi
+  elif [[ ${1} != "." && $1 != $2 ]]; then
+    find_package_json $(dirname $1) $2
+  fi
+}
+
+GIT_REPO=$(git remote get-url origin)
+if [[ -z ${DEFAULT_BRANCH} ]]; then
+  DEFAULT_BRANCH=$(
+    git ls-remote --symref ${GIT_REPO} HEAD 2>stderr \
+      | awk '/^ref:/ {sub(/refs\/heads\//, "", $2); print $2}'
+    )
+fi
+
+git fetch origin ${DEFAULT_BRANCH}
```

Comment:
Rather than show this in the job output, redirect it to `/dev/null` since it isn't relevant context for the engineer who'll be reviewing why the workflow failed.